eslint-local-rules/disallowConstEnumExports.js

```diff
@@ -0,0 +1,67 @@
+const { SymbolFlags } = require('typescript')
+
+/**
+ * This rule forbids exporting 'const enums'.
+ *
+ * 'const enums' are useful in TS to optimize the generated code, but they don't produce any JS code
+ * on their own[0].  So, exporting them in public packages entry points should be avoided, because
+ * we don't know how public packages are used: it could be used by other builders like babel which
+ * don't have access to the type definition, so won't be able to access the enum value.
```

Comment:
FMU, the goal of `const enum` is to have the type info but not the js structure, so the behavior kinda make sense but seems really confusing from a client perspective. 
Not sure what was the original intention for `MediaInteractions` though.

Since we already had an [issue with exported enums](https://github.com/DataDog/browser-sdk/pull/630), should we not prevent enum exports?